pool_automation/configure.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Check inventory directory
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - include_tasks: tasks/check_inventory.yml
+      when: not (skip_inventory_check|default(false)|bool)
```

Comment:
This is repeated in 3 different playbooks, probably this better be moved to check_inventory.yml